site/content/docs/4.3/helpers/text-truncation.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
-title: Text
-description: Documentation and examples for common text utilities to control alignment, wrapping, weight, and more.
+title: Text truncation
+description: Quickly truncation the long text with an ellipsis. 
```

Comment:
```suggestion
description: Quickly truncate long strings of text with an ellipsis.
```

Reply:
I'd even remove "Quickly" ... doesn't really add anything meaningful here

Reply:
@mdo, @patrickhlauke Thanks for your review!